project/MicroServiceBuild.scala

```diff
@@ -0,0 +1,37 @@
+import play.core.PlayVersion
+import play.sbt.PlayImport._
+import sbt._
+
+object MicroServiceBuild extends Build with MicroService {
```

Comment:
`Build` is deprecated - All new projects are using `.sbt` format with `build.sbt`
look at `https://github.com/hmrc/help-to-save-integration-tests/blob/master/build.sbt`

Reply:
Ok I have created a new build.sbt file in the main project root, and taken a stab at what I think should go in there. Will delete MicroServiceBuild if and when its good to do so.